### PR TITLE
[ja] Translate tasks/administer-cluster/migrating-from-dockershim/find-out-runtime-you-use into Japanese

### PR DESCRIPTION
**Pull request by ptux**
Address #31254

### CODE REVIEW

File: content/ja/docs/tasks/administer-cluster/migrating-from-dockershim/find-out-runtime-you-use.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+<!-- overview -->
+
+このページでは、クラスター内のノードが使用している[コンテナランタイム](/docs/setup/production-environment/container-runtimes/)を確認する手順を概説しています。
+クラスターの実行方法によっては、ノード用のコンテナランタイムが事前に設定されている場合と、設定する必要がある場合があります。
+マネージドKubernetesサービスを使用している場合、ノードに設定されているコンテナランタイムを確認するためのベンダー固有の方法があるかもしれません。
+このページで説明する方法は、`kubectl` の実行が許可されていればいつでも動作するはずです。
+
+## {{% heading "prerequisites" %}}
+
+`kubectl`をインストールし、設定します。詳細は[ツールのインストール](/docs/tasks/tools/#kubectl)の項を参照してください。
```
Comment (riita10069) [inline content/ja/docs/tasks/administer-cluster/migrating-from-dockershim/find-out-runtime-you-use.md:16]:
```suggestion
`kubectl`をインストールし、設定します。詳細は[ツールのインストール](/ja/docs/tasks/tools/#kubectl)の項を参照してください。
```

File: content/ja/docs/tasks/administer-cluster/migrating-from-dockershim/find-out-runtime-you-use.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+content_type: task
+weight: 10
+---
+
+<!-- overview -->
+
+このページでは、クラスター内のノードが使用している[コンテナランタイム](/docs/setup/production-environment/container-runtimes/)を確認する手順を概説しています。
+クラスターの実行方法によっては、ノード用のコンテナランタイムが事前に設定されている場合と、設定する必要がある場合があります。
+マネージドKubernetesサービスを使用している場合、ノードに設定されているコンテナランタイムを確認するためのベンダー固有の方法があるかもしれません。
+このページで説明する方法は、`kubectl` の実行が許可されていればいつでも動作するはずです。
```
Comment (riita10069) [inline content/ja/docs/tasks/administer-cluster/migrating-from-dockershim/find-out-runtime-you-use.md:12]:
```suggestion
このページで説明する方法は、`kubectl`の実行が許可されていればいつでも動作するはずです。
```

**Review by Arhell — COMMENTED** [review]
/assign @atoato88

File: content/ja/docs/tasks/administer-cluster/migrating-from-dockershim/find-out-runtime-you-use.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
+---
+title: ノードで使用されているコンテナランタイムの確認
+content_type: task
+weight: 10
+---
+
+<!-- overview -->
+
+このページでは、クラスター内のノードが使用している[コンテナランタイム](/docs/setup/production-environment/container-runtimes/)を確認する手順を概説しています。
+クラスターの実行方法によっては、ノード用のコンテナランタイムが事前に設定されている場合と、設定する必要がある場合があります。
```
Comment (atoato88) [inline content/ja/docs/tasks/administer-cluster/migrating-from-dockershim/find-out-runtime-you-use.md:10]: [Original text](https://github.com/kubernetes/website/blame/main/content/en/docs/tasks/administer-cluster/migrating-from-dockershim/find-out-runtime-you-use.md#L13) has empty row here.
Could you add empty row here?
```suggestion

クラスターの実行方法によっては、ノード用のコンテナランタイムが事前に設定されている場合と、設定する必要がある場合があります。
```

**Comment by atoato88** [comment]
/lgtm
Thank you to quick fix!

**Pull request merged**